server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/impl/camel/MatcherSplitter.java

```diff
@@ -120,7 +120,9 @@
                     ProcessorUtil.verifyMailAddresses(matchedRcpts);
                 }
 
-            } catch (MessagingException me) {
+            } catch (IOException e) {
```

Comment:
What is specific about IOException? Why can't it be handled as a normal exception?

Reply:
I'm not sure what is specific about IOException. The original code handles it differently, so I left it as it is, but I think onMatchException setting should be honored in the same way when IOException occurs, so I will remove it.